docs/docs/development/docs.md

```diff
@@ -0,0 +1,16 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Documentation
+
+Write documentation if necessary so that developer, administrators and users have an easily accessable way of learning about the collaboration manager.
```

Comment:
```suggestion
Write documentation if necessary so that developer, administrators and users have an easily accessible way of learning about the collaboration manager.
```